uint8 no_detected_person
uint8 no_deleted_detection
float32[] person_xyz
